pytradfri/__main__.py

```diff
@@ -3,8 +3,8 @@
 
 import argparse
 import logging
-from pprint import pprint
 import uuid
+from pprint import pprint
```

Comment:
Can we add isort configuration to pyproject.toml to avoid changing the sorting?

Reply:
I think we may just need to make sure that ruff picks up the existing isort config?

Reply:
> Can we add isort configuration to pyproject.toml to avoid changing the sorting?

Yes, I'll add that.

Reply:
I thought ruff would extend config automatically. Maybe there's a flag for that?

Reply:
I did some digging in ruff's docs and GitHub issues section and couldn't find any obvious solutions but adding a separate section.

Reply:
Ok.